Sources/SwiftKafka/KafkaConfig.swift

```diff
@@ -80,6 +82,41 @@ public struct KafkaConfig: Hashable, Equatable {
             return .init(pointer: duplicatePointer)
         }
 
+        /// Message callback has to be instantiated before KafkaClient is initialized
+        /// as pointer to rd_kafka_conf_t gets freed after rd_kafka_new()
+        func setMessageCallback() {
+            rd_kafka_conf_set_dr_msg_cb(
+                self.pointer,
+                { kafkaHandle, message, _ in
+
+                    guard let kafkaHandle = kafkaHandle, let message = message else {
+                        return // TODO: log error
```

Comment:
I would like to log an error here, however, we cannot capture values in C closures. Do you have any idea how we could access the `logger` of the `KafkaProducer` from this closure?